style.css

```diff
@@ -1,15 +1,641 @@
 }
 
 body {
-    color: #fff;
-    margin: 0;
-    padding: 0;
+  color: #fff;
+  margin: 0;
+  padding: 0;
   font-family: Roboto, sans-serif;
   font-style: normal;
+}
+
+.header-container {
+  margin-top: 15px;
+}
+
+.header-skills {
+  color: #ec5242 !important;
+  font-family: cocogoose, sans-serif;
+  font-weight: 200;
+  font-size: 24px;
+}
+
+h1.header-skills {
+  font-size: 21px;
+}
+
+h2.header-3 {
+  font-size: 25px;
+}
+
+.header-3 {
+  color: #ec5242 !important;
+  font-family: cocogoose_i, sans-serif !important;
+  font-weight: 900 !important;
+  font-size: 2rem;
+  text-transform: uppercase;
+}
+
+.bars {
+  top: 0;
+  bottom: 0;
+  color: darkgray;
+}
+
+h3.description-paragraph {
+  font-size: 16px;
+  font-weight: 500;
+  border: 2px solid #fff;
+  padding: 8px;
+  font-family: Lato, sans-serif;
+}
+
+.welcome-container {
+  background-image: url(images/main-background.png);
+}
+
+.web-image {
+  width: 100%;
+}
+
+.services-container {
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url(images/services.png);
+  margin: 32px 0;
+}
+
+h2.our-services {
+  font-size: 21px;
+}
+
+.flex-header {
+  color: white;
+  font-family: cocogoose_l, sans-serif;
+  font-weight: bold;
+  font-size: 1.3rem;
+  text-align: center;
+}
+
+.view-courses {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.services {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  padding: 0 8px;
+  margin: 16px;
+  background-color: rgba(57, 60, 67, 0.7);
+  border-radius: 16px;
+}
+
+.card-img {
+  width: 48px;
+  height: 48px;
+}
+
+.card-title {
+  font-size: 14px;
+  font-family: cocogoose, sans-serif;
+  color: #ec5242 !important;
+  padding: 18px 21px;
+  font-weight: 600;
+}
+
+.card-text {
+  font-size: 10px;
+  color: #fff;
+  padding-top: 8px;
+}
+
+.line-1 {
+  width: 20%;
+  border-color: #ec5242 !important;
+}
 
-}
\ No newline at end of file
+.join-container {
+  margin-bottom: 32px;
+}
+
+.btn-join {
+  height: 60px;
+  width: 250px;
+  background: #ec5242;
+  color: #fff;
+  font-family: cocogoose_l, sans-serif;
+  font-size: 1em;
+  border: none;
+  align-self: center;
+  margin: 2rem 0;
+  border-radius: 1rem;
+}
+
+.show-items {
+  transform: translateY(0) !important;
+  width: 100%;
+}
+
+.nav-items {
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 99;
+  background-color: rgb(60, 58, 57, 0.95);
+  color: white;
+  list-style: none;
+  padding-top: 4rem;
+
+  /* margin: 0 auto; */
+  width: 100%;
+}
+
+.hamburger-container {
+  position: relative;
+  z-index: 100;
+  top: 1rem;
+  right: 1rem;
+  padding: 4px;
+  cursor: pointer;
+  font-size: 32px;
+}
+
+i.fa.fa-times.close-icon {
+  display: none;
+  position: fixed;
+  padding: 15px 75% 15px 75%;
+  font-size: 24px;
+}
+
+i.fa.fa-bars {
+  position: fixed;
+}
```

Comment:
- [ ] [Optional] Kindly remove the fixed position of the hamburger button to make the design more responsive.